app/assets/javascripts/charges.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
file deleted from the repository https://github.com/fabriziobertoglio1987/railsmentor/commit/36c8a41b58caa421d02225ca4ef8d03344c32cdb